cowprotocol/cowswap
# fix(wc): add back patch which fixes WC path issue

# Summary

Fixes latest release issue with WC.

We did not notice but, [this file](https://github.com/cowprotocol/cowswap/pull/3226/files#diff-10b33a8e26a4a23da5a8b5f3cd5c51ffa96b996619c18d2242632189f225ffdd) was removed by mistake in the last PR part of the release.
And I failed to check WC on barn and staging.

  # To Test

1. Open the app
2. Connect with WC v2
* Should succeed
3. Place some orders
* Should work as expected
4. Try to connect other WC based wallets, such as Ambire, Trust and Alpha
* All of them should connect as expected
